wcfsetup/install/files/lib/data/blacklist/entry/BlacklistEntry.class.php

```diff
@@ -24,9 +24,9 @@ class BlacklistEntry extends DatabaseObject {
 	 * @param string $username
 	 * @param string $email
 	 * @param string $ipAddress
-	 * @return boolean
+	 * @return string[]
 	 */
-	public static function shouldReject($username, $email, $ipAddress) {
+	public static function getMatches($username, $email, $ipAddress) {
 		if (BLACKLIST_SFS_USERNAME === 'skip' && BLACKLIST_SFS_EMAIL_ADDRESS === 'skip' && BLACKLIST_SFS_IP_ADDRESS === 'skip') {
 			return false;
```

Comment:
Shouldn't this return `[]` then to match the function signature.

Reply:
Good catch, thanks!